tfhe/src/core_crypto/algorithms/lwe_bootstrap_key_conversion.rs

```diff
@@ -2,17 +2,58 @@
 //! [`standard LWE bootstrap keys`](`LweBootstrapKey`) to various representations/numerical domains
 //! like the Fourier domain.
 
+use super::ggsw_conversion::convert_standard_ggsw_ciphertext_to_fourier_mem_optimized;
 use crate::core_crypto::commons::computation_buffers::ComputationBuffers;
+use crate::core_crypto::commons::math::fft::{
+    par_convert_polynomials_list_to_fourier, Fft, FftView,
+};
 use crate::core_crypto::commons::traits::*;
+use crate::core_crypto::commons::utils::izip;
+use crate::core_crypto::entities::fourier_lwe_bootstrap_key::{
+    FourierLweBootstrapKey, FourierLweBootstrapKeyMutView,
+};
 use crate::core_crypto::entities::*;
 use crate::core_crypto::fft_impl::fft128::crypto::bootstrap::Fourier128LweBootstrapKey;
 use crate::core_crypto::fft_impl::fft128::math::fft::Fft128;
-use crate::core_crypto::fft_impl::fft64::crypto::bootstrap::{
-    fill_with_forward_fourier_scratch, FourierLweBootstrapKey,
-};
-use crate::core_crypto::fft_impl::fft64::math::fft::{Fft, FftView};
 use concrete_fft::c64;
-use dyn_stack::{PodStack, SizeOverflow, StackReq};
+use dyn_stack::{PodStack, ReborrowMut, SizeOverflow, StackReq};
+
+impl<'a> FourierLweBootstrapKeyMutView<'a> {
```

Comment:
should this be in the entities module for now ?

My reasoning : the algorithm for now is tied to the entity, what we would do is the same you did in the ggsw_conversion module i.e. rewrite the algorithm in this file as a function not in an impl block ?